group-coordinator/src/main/java/org/apache/kafka/coordinator/group/ReplicatedGroupCoordinator.java

```diff
@@ -239,6 +283,14 @@ public void replay(Record record) throws RuntimeException {
         ApiMessageAndVersion value = record.value();
 
         switch (key.version()) {
+            case 0:
```

Comment:
Do we expect this for keys 0 and 1?

Reply:
yeah, i assume that we were going to use the value record version only for offset commits then we started adding other record types

Reply:
Yeah, this is actually correct. Weird but correct.